index.js

```diff
@@ -1,5 +1,9 @@
-const minimist = require('minimist');
+const Input = require('./lib/Input');
+const Note = require('./lib/notes');
 
-console.log(minimist(process.argv))
+let input = new Input(process.argv);
 
+const isValid = input.valid();
+// Input(notes) ? Note(notes) : console.log('Bad Input');
 
+isValid ? Note.execute(input) : console.log('Fail');
```

Comment:
files should end with a newline. you can configure vs code to add it for you automatically